apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx

```diff
@@ -45,9 +83,15 @@ export function AccountSubHeaderDrawer({ isOpen, closeDrawer, family, team }: Pr
         </Box>
         <Box py={2}>
           <Description>
-            {t("account.subHeader.drawer.description3", {
-              team,
-            })}
+            {teamLink ? (
+              <TeamLink team={team} teamLink={teamLink} />
+            ) : (
+              <>
```

Comment:
you don't need a fragment here, simply the 
```
t("account.subHeader.drawer.description3", {
                  team,
                })
```

Reply:
Done in https://github.com/LedgerHQ/ledger-live/pull/8028/commits/b804236dd4d8d8f24b41a6defeb592fe84d8172b 👍